composer.json

```diff
@@ -48,5 +50,12 @@
         "cs": "phpcs",
         "cs-fix": "phpcbf",
         "test": "phpunit"
+    },
+    "extra": {
+        "zf": {
```

Comment:
So basically we no longer need to do that : https://github.com/maestrooo/shopify-sms-notifications/blob/master/php-app/config/config.php#L19 ?